streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -365,11 +368,14 @@ public static StreamThread create(final InternalTopologyBuilder builder,
         log.info("Creating consumer client");
         final String applicationId = config.getString(StreamsConfig.APPLICATION_ID_CONFIG);
         final Map<String, Object> consumerConfigs = config.getMainConsumerConfigs(applicationId, getConsumerClientId(threadId), threadIdx);
+        consumerConfigs.put(InternalConfig.TIME, time);
         consumerConfigs.put(StreamsConfig.InternalConfig.TASK_MANAGER_FOR_PARTITION_ASSIGNOR, taskManager);
         consumerConfigs.put(StreamsConfig.InternalConfig.STREAMS_METADATA_STATE_FOR_PARTITION_ASSIGNOR, streamsMetadataState);
         consumerConfigs.put(StreamsConfig.InternalConfig.STREAMS_ADMIN_CLIENT, adminClient);
         final AtomicInteger assignmentErrorCode = new AtomicInteger();
         consumerConfigs.put(StreamsConfig.InternalConfig.ASSIGNMENT_ERROR_CODE, assignmentErrorCode);
+        final AtomicLong nextProbingRebalanceMs = new AtomicLong(Long.MAX_VALUE);
+        consumerConfigs.put(StreamsConfig.InternalConfig.NEXT_PROBING_REBALANCE_MS, nextProbingRebalanceMs);
```

Comment:
This feels a bit out of scope, since we don't currently have _any_ tests that verify `StreamThread#create()`. Also, failing to set this attribute would instantly break all of our integration tests, since the consumer wouldn't be able to instantiate the assignor.

Not saying it wouldn't be valuable to have a unit test for it, but maybe we can call this one a follow-on task.

Reply:
Agreed